components/editor/NoteHeader.tsx

```diff
@@ -39,34 +39,52 @@ export default function NoteHeader() {
     (state) => !state.isSidebarOpen && state.openNoteIds?.[0] === currentNote.id
   );
   const isCloseButtonVisible = useStore(
-    (state) => state.openNoteIds?.[0] !== currentNote.id
+    (state) => state.openNoteIds?.[1]
```

Comment:
We want to show the close button if there is more than 1 open note. It's better to make this check more explicit by checking `state.openNoteIds.length > 1`